astro/src/content/docs/_shared/_userinfo-response.mdx

```diff
@@ -30,72 +24,42 @@ import InlineField from 'src/components/InlineField.astro';
-  </APIField>
-  <APIField name="family_name" type="String" renderif={!props.show_since}>
+  <APIField name="family_name" type="String" since="1.1.0" show_since={props.show_since}>
     The last name of the User if available.
 
     In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
   </APIField>
-  <APIField name="given_name" type="String" since="1.1.0" renderif={props.show_since}>
-    The first name of the User if available.
-
-    In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
-  </APIField>
-  <APIField name="given_name" type="String"  renderif={!props.show_since}>
+  <APIField name="given_name" type="String" since="1.1.0" show_since={props.show_since}>
     The first name of the User if available.
 
     In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
   </APIField>
-  <APIField name="name" type="String" since="1.1.0" renderif={props.show_since}>
+  <APIField name="name" type="String" since="1.1.0" show_since={props.show_since}>
     The full name of the User if available.
 
     In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
   </APIField>
-  <APIField name="name" type="String" renderif={!props.show_since}>
-    The full name of the User if available.
-
-    In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
-  </APIField>
-  <APIField name="middle_name" type="String" since="1.1.0" renderif={props.show_since}>
-    The middle name of the User if available.
-
-    In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
-  </APIField>
-  <APIField name="middle_name" type="String" renderif={!props.show_since}>
+  <APIField name="middle_name" type="String" since="1.1.0" show_since={props.show_since}>
     The middle name of the User if available.
 
     In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
   </APIField>
-  <APIField name="phone_number" type="String" since="1.1.0" renderif={props.show_since}>
-    The phone number of the User if available.
+  <APIField name="phone_number" type="String" since="1.50.0" show_since={props.show_since}>
```

Comment:
Why did this change to `since="1.50.0"`? The phone_number value has been around longer, its handling just changed in `1.50.0`. Can we keep the since of `1.1.0` and leave the old blurb about 
> In version `1.50.0` and later, ...

Reply:
I'm mistaken. I thought because the service was created in 1.50.0 that's when the endpoint was created so I thought the docs were wrong, but they are not. Pushed a revert.